### DISCUSSION TITLE
ability to float panel with margin settings?

### DISCUSSION THREAD

**Discussion by martinlindelof** [home-sweet-gnome/dash-to-panel#2025] [Ideas]
Didn't find any floating discussion, wonder if it's possible. As I first saw in Xfce, then in Deepin and now in Kde beta6. The ability to float the panel/dock and set margins. It could be optional under settings Position, then under Panel Length % there could be an Panel Offset or Margin setting, maybe also in % so you can match the length settings to get an even margin around the panel.

**Comment by DSDV** [comment]
I have to dig this up
i also recently saw a system with a floating bar and especially for folks with highdpi widescreens where the panel is not 100% of the monitor with a margin with intellihide would be a really great feature (:
